package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "design-system",
-  "version": "0.14.4",
+  "version": "0.14.5”,
```

Comment:
There are visual differences. The reference page is completely different even though all I did was update the SVGs.